backend/app/services/prompt_optimization.py

```
@@ -0,0 +1,302 @@
+
+    if not team_name:
+        raise ValueError(f"Task {task_id} has no teamRef")
+
+    team = (
+        db.query(Kind)
+        .filter(
+            Kind.kind == "Team",
+            Kind.name == team_name,
+            Kind.namespace == team_namespace,
+            Kind.user_id == user_id,
+            Kind.is_active == True,
```

Ruff (0.15.9)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)